charts/lily/templates/configmap.yaml

```diff
@@ -0,0 +1,42 @@
+---
+{{- if .Values.daemon.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-lily-config
+  labels:
+    {{- include "sentinel-lily.labels" . | nindent 4 }}
+data:
+  config.toml: |
+    [API]
+      ListenAddress = "/ip4/127.0.0.1/tcp/1234/http"
+    [Libp2p]
+      ListenAddresses = ["/ip4/0.0.0.0/tcp/1347"]
+      ConnMgrLow = 400
+      ConnMgrHigh = 500
+      ConnMgrGrace = "5m0s"
+    {{- if .Values.daemon.pubsub.ipwhitelist }}
+    [Pubsub]
+      IPColocationWhitelist = ["{{ join "\", \"" .Values.daemon.pubsub.ipwhitelist }}"]
+    {{- end }}
+    [Storage]
+      {{- if .Values.daemon.storage.postgresql }}
+      [Storage.Postgresql]
+        {{- range .Values.daemon.storage.postgresql }}
+        [Storage.Postgresql.{{ .name }}]
+          SchemaName = {{ .schema | default "lily" | quote }}
+          URLEnv = "LILY_STORAGE_POSTGRESQL_{{ .name | upper }}_URL"
+          ApplicationName = {{ .applicationName | default (include "sentinel-lily.instanceName" $ ) | quote }}
+          PoolSize = {{ .poolSize | default 20 }}
+          AllowUpsert = {{ .allowUpsert | default false }}
+        {{- end }}
+      {{- end }}
+      {{- if .Values.daemon.storage.file }}
+      [Storage.File]
```

Comment:
We don't need the file storage here

Reply:
As a generic configuration script, wouldn't it be appropriate to offer all of the features? Or are you suggesting we don't need it for another reason?